components/ui/workspacesBar/WorkspaceBar.tsx

```diff
@@ -59,6 +74,132 @@ function WorkspaceBar({
+                  <svg
+                    xmlns="http://www.w3.org/2000/svg"
+                    fill="none"
+                    viewBox="0 0 24 24"
+                    strokeWidth="1.5"
+                    stroke="currentColor"
+                    className="w-6 h-6"
+                  >
+                    <path
+                      strokeLinecap="round"
+                      strokeLinejoin="round"
+                      d="M18 7.5v3m0 0v3m0-3h3m-3 0h-3m-2.25-4.125a3.375 3.375 0 1 1-6.75 0 3.375 3.375 0 0 1 6.75 0ZM3 19.235v-.11a6.375 6.375 0 0 1 12.75 0v.109A12.318 12.318 0 0 1 9.374 21c-2.331 0-4.512-.645-6.374-1.766Z"
+                    />
+                  </svg>
+
+                  <span className="sr-only">Refresh icon</span>
+                </div>
+                {currentNotification[currentSessionInUse] &&
+                  currentNotification[currentSessionInUse].map(
+                    (notification) => {
+                      return (
+                        <div className="ms-3 text-sm font-normal">
+                          <span className="mb-1 text-sm font-semibold text-gray-900 dark:text-white">
+                            Friend request
+                          </span>
+                          <div className="mb-2 text-sm font-normal">
+                            User {notification.peer_cid.value} wants to be
+                            friends with you
+                          </div>
+                          <div className="grid grid-cols-2 gap-2">
+                            <div>
+                              <button
+                                onClick={() => {
+                                  peerRegister({
+                                    cid: currentSessionInUse,
+                                    peerCid: notification.peer_cid.value,
+                                  });
+                                  dispatch(
+                                    deleteFromNotificationsContext({
+                                      cid: currentSessionInUse,
+                                      peerCid: notification.peer_cid.value,
+                                    })
+                                  );
+                                }}
+                                className="inline-flex justify-center w-full px-2 py-1.5 text-xs font-medium text-center text-white bg-blue-600 rounded-lg hover:bg-blue-700 focus:ring-4 focus:outline-none focus:ring-blue-300 dark:bg-blue-500 dark:hover:bg-blue-600 dark:focus:ring-blue-800"
+                              >
+                                Accept
+                              </button>
+                            </div>
+                            <div>
+                              <span
+                                onClick={() => {
+                                  console.log('Not now');
```

Comment:
What is this "not now" print for? We should cleanup our debugging prints, if this is one of those cases.